src/test/java/org/openmrs/performance/registries/DoctorRegistry.java

```diff
@@ -94,6 +94,14 @@ public ChainBuilder openLabResultsTab(String patientUuid) {
 	public ChainBuilder openAllergiesTab(String patientUuid) {
 		return exec(httpService.getAllergies(patientUuid));
 	}
+
+	public ChainBuilder addAllergies(String patientUuid) {
+		return exec(
+			httpService.searchPatient("test location test"),
+			httpService.getAllergies(patientUuid),
```

Comment:
This http call is to get recorded allergies of a patient, we should fetch allergens instead. Check my other comment.